compiler/powershell.vim

```diff
@@ -29,14 +29,36 @@ endif
 
 if !executable(g:ps1_makeprg_cmd)
   echoerr "To use the powershell compiler, please set g:ps1_makeprg_cmd to the powershell executable!"
+  finish
```

Comment:
Why not [autodetect](https://github.com/PProvost/vim-ps1/blob/26a75886caef937bfad4201d5478571a992984c2/ftplugin/ps1.vim#L33) the powershell executable?

Reply:
It does directly above.